Content.Server/_Starlight/UXN/UXNDevice.cs

```diff
@@ -0,0 +1,810 @@
+using System.Collections;
+using Content.Shared._Starlight.UXN.Devices;
+namespace Content.Shared._Starlight.UXN;
+
+public sealed class Byte256
+{
+    private readonly byte[] _inner = new byte[256];
+
+    public Byte256() =>
+        Array.Fill<byte>(_inner, 0x00);
+
+    public byte this[int i]
+    {
+        get => _inner[i];
+        set => _inner[i] = value;
+    }
+
+    public byte[] ToRaw() => (byte[])_inner.Clone();
+}
```

Comment:
```suggestion
[InlineArray(256)]
public struct InlineByte256
{
    private byte _element0;
}
}
```

There is a chance that the sandbox will not allow this.
Still, try it that way.

Honestly, no idea how netcode will react to this.
No clue how easily this could turn into a way to completely wreck the server.

Reply:
I tried InlineArray initially. Sandbox didn't like it hence this hacky _inner setup